integration/bench/load_test.go

```diff
@@ -0,0 +1,78 @@
+package load
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	util "github.com/grafana/tempo/integration"
+
+	cortex_e2e "github.com/cortexproject/cortex/integration/e2e"
+	cortex_e2e_db "github.com/cortexproject/cortex/integration/e2e/db"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	k6Image = "loadimpact/k6:latest"
+)
+
+func TestAllInOne(t *testing.T) {
+	s, err := cortex_e2e.NewScenario("tempo_e2e")
+	require.NoError(t, err)
+	defer s.Close()
+
+	minio := cortex_e2e_db.NewMinio(9000, "tempo")
+	require.NotNil(t, minio)
+	require.NoError(t, s.StartAndWaitReady(minio))
+
+	require.NoError(t, util.CopyFileToSharedDir(s, "config.yaml", "config.yaml"))
+	require.NoError(t, util.CopyFileToSharedDir(s, "smoke_test.js", "smoke_test.js"))
+	require.NoError(t, util.CopyFileToSharedDir(s, "stress_test_write_path.js", "stress_test_write_path.js"))
+	require.NoError(t, util.CopyFileToSharedDir(s, "modules/util.js", "modules/util.js"))
+
+	tempo := util.NewTempoAllInOne()
+	require.NoError(t, s.StartAndWaitReady(tempo))
+
+	k6 := newK6Runner(tempo)
+	require.NoError(t, s.StartAndWaitReady(k6))
+
+	require.NoError(t, runK6Test(k6, "smoke_test.js"))
+	require.NoError(t, runK6Test(k6, "stress_test_write_path.js"))
+}
+
+func runK6Test(k6 *cortex_e2e.ConcreteService, testjs string) error {
+	fmt.Println("------ " + testjs + " ------")
+	stdout, stderr, err := k6.Exec(cortex_e2e.NewCommand("k6", "run", "--quiet", filepath.Join(cortex_e2e.ContainerSharedDir, testjs)))
+	fmt.Println("------ stdout ------")
+	fmt.Println(stdout)
+
+	if err != nil {
+		fmt.Println("------ stderr ------")
+		fmt.Println(stderr)
+	}
+
+	return err
+}
+
+func newK6Runner(tempo *cortex_e2e.HTTPService) *cortex_e2e.ConcreteService {
+	s := cortex_e2e.NewConcreteService(
+		"k6",
+		k6Image,
+		cortex_e2e.NewCommandWithoutEntrypoint("sh", "-c", "sleep 3600"),
```

Comment:
Do we need the sleep here? This will only start up once tempo is up and running, i.e after this 
```
	require.NoError(t, s.StartAndWaitReady(tempo))
```

Reply:
We just need a process to start and not exit.  If you run 
```
docker run loadimpact/k6:latest
```

Then it immediately exits.  So we start this k6 container with sleep so it just hangs around for awhile and then use exec:

https://github.com/grafana/tempo/pull/392/files/8f5c168ffb88850487e9932fc779df9da03caf8d#diff-4f32d9cc23405c3bef7b0d0233346dd096936a420e6499b8981bf851f995f0d8R45

to kick off our tests.